src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -982,7 +983,7 @@ class CreateProcedureStatementSegment(BaseSegment):
     https://docs.snowflake.com/en/sql-reference/sql/create-procedure.html
     """
 
-    type = "create_clone_statement"
```

Comment:
Is this a bug fix? It seems unrelated to the rest of the PR.

Reply:
Yes, kind of a drive by bug fix. I assume that the `CreateCloneSegment` was copied to create this segment (same as I copied this segment), but the author forgot to change this value.